src/mainboard/sunxi/nezha/README.md

```diff
@@ -37,8 +37,8 @@ and to be found within Linux in the directory `arch/riscv/boot/dts/allwinner/`.
 We currently do not patch it at runtime, so you need to hardcode your board's
 memory size into it.
 
-For a work-in-progress Linux 5.19 (not all patches are upstream yet), see:
-https://github.com/orangecms/linux/tree/5.19-smaeul-plus-dts
+Not all patches are upstream yet. [work-in-progress Linux 5.19](
+https://github.com/orangecms/linux/tree/5.19-smaeul-plus-dts/arch/riscv/boot/dts/allwinner)
```

Comment:
More than just the DTS is needed, so the reference was to the branch.

How do you feel about keeping the phrasing as is, adding the Markdown link formatting, and giving the link a title that is the URL again, so it'll be renderd in Rust docs just like on GitHub?
We could improve the wording thereafter.